tests/ragger/app/command_builder.py

```diff
@@ -168,3 +176,50 @@ def eip712_filtering_show_field(self, name: str, sig: bytes) -> bytes:
                                P1Type.COMPLETE_SEND,
                                P2Type.FILTERING_FIELD_NAME,
                                self._eip712_filtering_send_name(name, sig))
+
+    def sign(self, bip32_path: list[Optional[int]], data: list) -> Iterator[bytes]:
+        payload = self._format_bip32(bip32_path, bytearray())
+        payload += rlp.encode(data)
+        p1 = P1Type.SIGN_FIRST_CHUNK
+        while len(payload) > 0:
+            yield self._serialize(InsType.SIGN,
+                                  p1,
+                                  0x00,
+                                  payload[:0xff])
+            payload = payload[0xff:]
+            p1 = P1Type.SIGN_SUBSQT_CHUNK
+
+    def send_fund(self,
```

Comment:
I changed this, now the client directly calls the `sign` function of the command builder instead of having an intermediate `send_fund` function in the command builder which indeed made no sense. :+1: